schedule/yast/btrfs/btrfs_sle_libstorage-ng@hmc+ipmi.yaml

```diff
@@ -0,0 +1,96 @@
+name:           btrfs_libstorage-ng@hmc+ipmi
+description:    >
+  Validate default installation with btrfs and Libstorage NG.
+vars:
+  DESKTOP: gnome
+  FILESYSTEM: btrfs
+schedule:
+  # Called on BACKEND: qemu
+  - {{isosize}}
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  # Called only on BACKEND: s390x
+  - {{disk_activation}}
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_filesystem
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  # Called on all, except BACKEND: s390x, ipmi
+  - {{hostname_inst}}
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  # Called on BACKEND: s390x, svirt, ipmi
+  - {{reconnect_mgmt_console}}
+  # Called on BACKEND: qemu
+  - {{grub_test}}
+  - installation/first_boot
+  - console/validate_no_cow_attribute
+  # On all the backends except s390x, /home is located on a separate partition
+  - {{validate_home_partition}}
+conditional_schedule:
+  isosize:
+    BACKEND:
+      qemu:
+        - installation/isosize
+  disk_activation:
+    BACKEND:
+      s390x:
```

Comment:
could you please delete conditional modules not related with hmc or ipmi in this yaml.